pages/disability-benefits/apply/form-526-disability-claim.md

```diff
@@ -1,16 +0,0 @@
----
-title: Apply For Increased Disability Benefits
-heading: Apply for increased disability benefits
-display_title: File for increased disability
-entryname: 526EZ-claims-increase
-layout: page-react.html
-description: Learn how to apply online for increased disability compensation.
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/disability/">Disability benefits</a></li>
-    <li><a aria-current="page" href="/disability/how-to-file-claim/">File for a disability increase</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-9f1d0deef32d4274307fcec90c1bb5e2R5.

Breadcrumbs confirmed to look okay.